.golangci.reference.yml

```diff
@@ -467,12 +468,22 @@ linters-settings:
         # - 'all':    fail on all errors.
         # - 'import': ruleguard rule imports a package that cannot be found.
         # - 'dsl':    gorule file does not comply with the ruleguard DSL.
+        # Default: ""
         failOn: dsl
         # Comma-separated list of file paths containing ruleguard rules.
         # If a path is relative, it is relative to the directory where the golangci-lint command is executed.
         # The special '${configDir}' variable is substituted with the absolute directory containing the golangci config file.
         # Glob patterns such as 'rules-*.go' may be specified.
+        # Default: ""
         rules: '${configDir}/ruleguard/rules-*.go,${configDir}/myrule1.go'
+        # Comma-separated list of enabled groups or skip empty to enable everything.
+        # Tags can be defined with # character prefix.
+        # Default: "<all>"
+        enable: "badLock,httpNoBody,#experimental,#style"
+        # Comma-separated list of disabled groups or skip empty to enable everything.
+        # Tags can be defined with # character prefix.
+        # Default: ""
+        disable: "badLock,httpNoBody,#experimental,#style"
```

Comment:
You are using go-critic rules in the ruleguard section.
ruleguard allows running user-defined rules and doesn't interact with go-critic rules.
Can you change those values?